sourcecode/hub/resources/views/content/standalone-view.blade.php

```diff
@@ -1,71 +1,64 @@
-
-                    <button type="button" class="btn btn-primary mx-2">
-                        {{ trans('messages.more-details')}}
-                    </button>
-                </div>
+            <div class="buttons-container d-flex justify-content-center">
+                <button type="button" class="btn btn-secondary mx-2" data-bs-toggle="modal" data-bs-target="#reportModal" aria-label="{{ trans('messages.report-content')}}">
+                    {{ trans('messages.report-content')}}
+                </button>
+                <button type="button" class="btn btn-primary mx-2" aria-label="{{ trans('messages.more-details')}}">
+                    {{ trans('messages.more-details')}}
+                </button>
             </div>
         </div>
     </div>
-
     <!-- Report Content Modal -->
-    <div class="modal fade" id="reportModal" tabindex="-1" aria-labelledby="reportModalLabel" aria-hidden="true" data-bs-backdrop="static">
+    <div class="modal fade" id="reportModal" tabindex="-1" aria-labelledby="{{ trans('messages.report-modal-label')}}" aria-hidden="true" data-bs-backdrop="static">
         <div class="modal-dialog modal-dialog-centered modal-lg">
             <div class="modal-content">
                 <div class="modal-header border-0">
                     <h5 class="modal-title" id="reportModalLabel">{{ trans('messages.report-this-content')}}</h5>
-                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="{{ trans('messages.close')}}"></button>
                 </div>
                 <div class="modal-body border-0">
                     <p>{{ trans('messages.this-game-has-content')}}</p>
                     <div class="d-flex flex-wrap gap-2 mt-4">
-                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="Copyrighted">{{ trans('messages.copyrighted')}}</button>
-                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="Violent">{{ trans('messages.violent')}}</button>
-                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="Sexual">{{ trans('messages.sexual')}}</button>
-                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="Other">{{ trans('messages.other')}}</button>
+                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="{{ trans('messages.copyright')}}">{{ trans('messages.copyright')}}</button>
+                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="{{ trans('messages.violence')}}">{{ trans('messages.violence')}}</button>
+                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="{{ trans('messages.sexual')}}">{{ trans('messages.sexual')}}</button>
```

Comment:
The content of the element is a readable text, no need to repeat it using `aria-label`